src/test/obj_first_next/obj_first_next.c

```diff
@@ -305,7 +305,8 @@ main(int argc, char *argv[])
 		int list_num;
 		int id;
 		char type;
-		sscanf(argv[i], "%c:%d:%d", &type, &list_num, &id);
+		if (sscanf(argv[i], "%c:%d:%d", &type, &list_num, &id) == EOF)
```

Comment:
This is not the only error case. You should also handle the case in which sscanf was unable to read all of the inputs.